src/main/java/kostka/moviecatalog/service/DbMovieService.java

```diff
@@ -113,14 +114,14 @@ public void deleteMovie(final Long movieId) {
     }
 
     private Movie populateMovie(final MovieFormDto dto) {
-        Movie movie = new Movie();
-        movie.setName(dto.getName());
-        movie.setDescription(dto.getDescription());
-        movie.setDirector(dto.getDirector());
-        movie.setMusic(dto.getMusic());
-        movie.setCamera(dto.getCamera());
-        movie.setForAdults(dto.isForAdults());
-        return movie;
+        return new MovieBuilder()
```

Comment:
👍 Jako příklad builderu beru :) Tady myslím není nic, co vytvknout.

Pak si jen všimni, že kdyby si ze setterů v `Movie` vracel taky this a udělal to taky jako Fluent API, tady by ten zápis byl skoro stejný :)